test/CatapultECC.spec.ts

```diff
@@ -19,13 +19,14 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 import {expect} from "chai";
-import {array, convert, nacl_catapult, sha3Hasher} from 'nem2-library';
+import {Crypto} from 'nem2-sdk';
 
 // internal dependencies
 import {
     CatapultECC,
     Cryptography,
 } from "../index";
+import { from } from "rxjs";
```

Comment:
why is rxjs needed here ?

Reply:
my studio code, always auto insert rxjs.
will remove it.